nibabel/freesurfer/io.py

```diff
@@ -622,3 +624,53 @@ def _serialize_volume_info(volume_info):
             strings.append(
                 f'{key:6s} = {val[0]:.10g} {val[1]:.10g} {val[2]:.10g}\n'.encode('utf-8'))
     return b''.join(strings)
+
+def read_stats_file(file_path):
+    """Extracts stats from stats files except '*curv.stats' files
+
+    Parameters
+    ----------
+    file_path: str, required
+
+    Examples
+    --------
+    >>> stats_a2009, column_names = read_stats_file(r'lh.aparc.a2009s.stats')
```

Comment:
If we're going to have example code, it needs to run. We could drop an example file in `nibabel/tests/data/` and access it with

```Python
from nibabel.testing import test_data
stats_a2009 = read_stats_file(test_data(fname='lh.aparc.a2009s.stats'))
```